resources/metadata/protecodeExecuteScan.yaml

```diff
@@ -275,3 +275,14 @@ spec:
                 type: int
               - name: vulnerabilities
                 type: int
+      - name: reports
+        type: reports
+        params:
+          - filePattern: "**/toolrun_protecode_*.json"
+            type: protecode
+          - paramRef: reportFileName
```

Comment:
Cool, didn't know we can support that. Will also do the trick for another internal step.
Thanks!